### ISSUE TITLE
Example in README not working

### ISSUE THREAD

**Issue by Semigradsky** [visionmedia/superagent#640]
In [**plugins** section](https://github.com/visionmedia/superagent#plugins):

> Superagent is easily extended via plugins.
```js
var nocache = require('no-cache');
var request = require('superagent');
var prefix = require('superagent-prefix')('/static');

>prefix(request); // Prefixes *all* requests

>request
.get('/some-url')
.use(nocache) // Prevents caching of *only* this request
.end(function(err, res){
    // Do something
});
```

**Comment by defunctzombie** [comment]
Not working how?

**Comment by Semigradsky** [comment]
Looks like fixed in c279f59bd4a5485a05cefca167a2d318e5591591 and 21216bcaebee3c0258935501266a4bc0b5d1eed3
